frontend/src/App.tsx

```diff
@@ -26,8 +26,8 @@ const App = () => {
   return (
     <ThemeProvider theme={theme === 'light' ? lightTheme : darkTheme}>
       <GlobalStyles />
-      <Suspense fallback={<PageLoading />}>
-        <ErrorBoundary>
```

Comment:
NOOOO dont tell me this was the bug

Reply:
https://github.com/csesoc/Circles/commit/384c237ca51d3a812030a91dca4773a2ff4c1b87
😳

Reply:
lmao its official that every *single* commit to dev jared made that day introduced a bug

Reply:
lmaooo i actually dont know if this is the bug or not. but like i dont think it makes sense for the error boundary to be lazy loaded coz its suppose to catch errors. will probs have to monitor this still tbh